docs/react/quickstart.md

```diff
@@ -198,20 +197,17 @@ Here in our `IonContent`, we're adding an `IonList` and a much more involved `Io
 
 ```tsx
 <IonItem>
-  <IonCheckbox slot="start" />
-  <IonLabel>
+  <IonCheckbox labelPlacement="end" justify="start">
     <h1>Create Idea</h1>
     <IonNote>Run Idea by Brandy</IonNote>
-  </IonLabel>
+  </IonCheckbox>
   <IonBadge color="success" slot="end">
     5 Days
   </IonBadge>
 </IonItem>
 ```
 
-Item is important as it clearly shows the mix of React concepts and Web Component concepts. The first clear example of a React concept is self-closing tags for React Components in `IonCheckbox`. This is just a simpler way of writing components that do not contain any child content.
-
-From the Web Components side, we have a special attribute called `slot`. This is key for letting the `IonItem` know where to place the `IonCheckbox` when it renders. This is not a React API, but a web standards API.
+Item is important as it clearly shows the mix of React concepts and Web Component concepts. From the Web Components side, we have a special attribute called `slot`. This is key for letting the `IonItem` know where to place the `IonBadge` when it renders. This is not a React API, but a web standards API.
```

Comment:
I couldn't figure out a way to keep the idea of demonstrating a React-specific concept without it sounding forced. I'm open to also removing the "clearly shows a mix of React concepts and Web Component concepts" sentence; the Vue page doesn't have it, but I wasn't sure whether the idea was important to mention.

Reply:
I don't think there is a lot of value to stating that `Item` shows React concepts. We can somewhat assume a developer using React with Ionic is at least familiar with React at a basic level. 

The concept of slots is unfamiliar to a React developer. They either 'slot' content as children or use a property with a JSX binding to render additional mark-up in a targeted area. 

I'd be in favor of cutting this information down in the quick start to focus more on the `slot` specifics. Something similar to:

> Item uses a feature similar to `children` or JSX props in React, called [`slots`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/slot). This feature allows developers to let `IonItem` know where to render the `IonBadge` in the template. This is a web standards API that is used across many Ionic Framework components. 

What do you think?

Reply:
Sounds good to me! I tweaked things in https://github.com/ionic-team/ionic-docs/pull/3415/commits/eeed7748d37d2bc48465a33318a399fdd5aac600.